helper/Cargo.toml

```diff
@@ -10,12 +10,13 @@ repository = "https://github.com/Cuprate/cuprate/tree/main/consensus"
 
 [features]
 # All features off by default.
-default   = []
+default   = ["std"]
```

Comment:
Either the comment needs to be updated or `"std"` should be removed, if `crypto` needs `std` it should be `crypto = ["std", "..."]`.